com.leinardi.gkraken.json

```diff
@@ -0,0 +1,37 @@
+{
+  "app-id": "com.leinardi.gkraken",
+  "command": "gkraken",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.32",
```

Comment:
Have you tried bumping this to 3.34?

Reply:
Not yet, I'll test it now and update if works fine.

Reply:
Build on 3.34 fails with this error:
```
Starting build of com.leinardi.gkraken
========================================================================
Building module libdbusmenu in /home/leinardi/Workspace/gitlab/gkraken/.flatpak-builder/build/libdbusmenu-1
========================================================================
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /usr/bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking whether to enable maintainer-specific portions of Makefiles... no
checking whether NLS is requested... yes
./configure: line 3080: intltool-update: command not found
checking for intltool >= 0.35.0...  found
configure: error: Your intltool is too old.  You need intltool 0.35.0 or later.
Error: module libdbusmenu: Processo figlio uscito con codice 1
```

Reply:
intltool was removed from Freedesktop 19.08. It's available from shared-modules. Just add that before libappindicator.

Reply:
Ok, I had to update the shared modules and add the `dbus-glib-0.110.json` before the `libappindicator-gtk3-introspection-12.10.json`.